win32/status.html

```diff
@@ -242,6 +242,7 @@ <h1 id="status-header">Agent info</h1>
             <dt>Flush count</dt>        <dd>{{ forwarder['flush_count'] }}</dd>
             <dt>Queue size</dt>         <dd>{{ forwarder['queue_size'] }} bytes</dd>
             <dt>Queue length</dt>       <dd>{{ forwarder['queue_length'] }}</dd>
+            <dt>Transactions Rejected (too large)</dt>       <dd>{{forwarder['too_big_count']}}</dd>
```

Comment:
nitpick: could you add spaces between `{{`, `}}` and the var that's displayed (just for consistency)

Also, it's unrelated but while we're at it: could you add the `Transactions received` and `Transactions flushed` stats on this page too. Thanks!